src/Symfony/Component/Intl/Countries.php

```diff
@@ -170,8 +223,30 @@ public static function getAlpha3Names(?string $displayLocale = null): array
         return $alpha3Names;
     }
 
+    /**
+     * Sets the internal `withUserAssigned` flag, overriding the default `SYMFONY_INTL_WITH_USER_ASSIGNED` value.
+     *
+     * The ISO 3166/MA has received information that the CE Commission has allocated the alpha-2 user-assigned code "XK"
+     * to represent Kosovo in the interim of being recognized by the UN as a member state.
+     *
+     * Set `$withUserAssigned` to true to have `XK`, `XKK` and `983` available in the other functions of this class.
+     */
+    public static function setWithUserAssigned(bool $withUserAssigned): bool
```

Comment:
given that this is both a setter and a getter, would it make sense to merge the two added methods?

```suggestion
    public static function withUserAssigned(?bool $withUserAssigned = null): bool
```

and we always return the current value (after setting it when $withUserAssigned is not null)?